scripts/setup

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+PROJECT_NAME="eev4"
+
+# runs the given command as root (detects if we are root already)
+runAsRoot() {
```

Comment:
Why we need root?

is it for /etc/hosts?

Reply:
For that also we'll need root privileges, but that is not part of the install script.

We're installing the repo in `/opt/eev4` and symlinking `/opt/eev4/scripts/eev4` to `/usr/local/bin/eev4` to put it in PATH.
```
runAsRoot cp -r "$EE_TMP_REPO" "$EE_INSTALL_DIR"
runAsRoot ln -s "$EE_INSTALL_DIR/$EE_SCRIPT" "$EE_SCRIPT_INSTALL_PATH"
```